maquetado/index.html

```diff
@@ -10,24 +10,24 @@
 </head>
 <body>
   <div class="book-container row">
-    <img src="./assets/book-cover.png" class="book-cover"/>
-    <div class="column book-info-container">
-      <div class="row book-title-info-container">
-        <h1 class="book-title bold height">Título del libro</h1>
-        <h2 class="book-gender height">(género)</h2>
-      </div>
-      <div class="row">
-        <h3 class="book-detail-name bold height">Autor del libro:</h3>
-        <h3 class="book-detail-value height">Nombre del autor del libro</h3>
-      </div>
-      <div class="row">
-        <h3 class="book-detail-name bold height">Editorial:</h3>
-        <h3 class="book-detail-value height">Nombre de la editorial</h3>
-      </div>
-      <div class="row">
-        <h3 class="book-detail-name bold height">Año de publicación:</h3>
-        <h3 class="book-detail-value height">Año de publicación</h3>
-      </div>
+        <img src="./assets/book-cover.png" class="book-cover">
```

Comment:
Delete the extra indent here: 

```suggestion
      <img src="./assets/book-cover.png" class="book-cover">
```